types/ioredis-mock/ioredis-mock-tests.ts

```diff
@@ -69,8 +69,8 @@ const f = async () => {
     // now you can see we have one new message
 
     // use xread to read all messages in channel
-    const messages1 = await redis.xread(['STREAMS', channel, 0]);
-    const messages2 = messages1[0][1];
+    const messages1 = await redis.xread('STREAMS', channel, 0);
```

Comment:
Confusingly, the ioredis examples don't compile using their own types: https://github.com/luin/ioredis/blob/308017a6b9429c16b074e03e70f5524499476fa9/examples/redis_streams.js#L28